### ISSUE TITLE
FE | Profile | Military information | 508 Defect 2, Screen Readers, "military information" headings nested improperly

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#28961]
Label: 28306

    - [508-defect-2]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2

Issue Title: 508 Defect 2, Screen Readers, "military information" headings nested improperly
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect headings to be nested in logical order.
<h2>Details</h2>
On the Military Information page, the first heading, "Profile" is an h4. This is followed by an H1, "Military Information." In the case of test user 1, the h1 is followed by an h3, "We can't access your military information." 
<h2>Acceptance Criteria</h2>
Headings should be nested properly.

<h2>Proposed Solution (if known)</h2>
Structure headings as though you were writing a term paper. The first heading on any page should always be H1, followed by H2, H3, etc. You can skip levels, for instance, from h4 to h2 in the same way a small subsection can be followed by a main section. Levels are never skipped going the other way.

**Comment by andaleliz** [comment]
@mtcA6 I just found this issue - unsure if it's already been addressed, but would be good for us to look at and evaluate. Anyone on the team could test with voiceover or look at the DOM.

**Comment by mtcA6** [comment]
@pamela-drouin @adamwhitlock1 @andaleliz should profile read w/ a 2 there? Is that ok? 

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/a31f9c38-6b57-47a7-bc3a-e9c55e873343)

Text from the image:
Military Information | Veterans Affairs
    2 Profile menu
1 Military information
    2 We can’t match your information to any military service records
        3 Request your military records (DD214)
    2 Veteran programs and services
    2 More VA resources
    2 Get VA updates
    2 In crisis? Talk to someone now
    2 Get answers
    2 Call us
    2 Visit a medical center or regional office
    2 Language assistance

Otherwise, it looks like the other headings mentioned were fixed.

**Comment by andaleliz** [comment]
Yes, I think that's ok. This is a problem that is the part of the hubless spoke problem that was not resolved the hub.  It's actually the main description in the[ hub epic.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40593) The links to the various sections refers to the Profile menu referenced at the top of the list above.

>The Accessibility team informed us that the links to the various sections of the Profile are above the H1 making it difficult for screenreaders to interpret and read the page structure correctly. See [slack](https://dsva.slack.com/archives/C909ZG2BB/p1651153191682419) thread. When a screenreader user opens the page, unless they know to navigate 'upwards' in the page structure, they will likely never realize the navigation panel on the left of the screen exists allowing them to access different features of the profile.

**Comment by mtcA6** [comment]
Starting from the hub it's resolved, when navigating w/in the actual sub pages it reverts to the old problem :/
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/86093938-3d00-41ed-b0d4-f4a50042032d)

**Comment by andaleliz** [comment]
@mtcA6 yep, that's correct :/
